client/me/account-password/index.jsx

```diff
@@ -140,7 +140,7 @@ const AccountPassword = createReactClass( {
 		const failure = head( this.props.accountPasswordData.getValidationFailures() );
 
 		if ( this.props.accountPasswordData.passwordValidationSuccess() ) {
-			return <FormInputValidation text={ translate( 'Your password can be saved.' ) } />;
+			return <FormInputValidation text={ translate( 'Strong Password.' ) } />;
```

Comment:
Just personal preference but I would prefer `Your password is strong enough to be saved`, my reasoning being that I doubt most passwords are actually *that* strong, but just strong enough to meet the requirements. 

```suggestion
			return <FormInputValidation text={ translate( 'Your password is strong enough to be saved.' ) } />;
```

Definitely not a big deal though!

Reply:
I'm totally fine with that. In fact it also goes with the message that we show if the password doesn't meet the criteria. I mean in terms of the words that we use. I will wait for @sixhours to take a look at it once :)

Reply:
I lean toward the simpler "Strong password"; it's more concise, and we state above that "Strong passwords have at least six characters, and use upper and lower case letters, numbers, and symbols like ! ” ? $ % ^ & )." By our definition, anything that meets that criteria is a strong password.

That said, we use sentences for all the other messages, so 🤷‍♀ . I think either is an improvement in clarity.

Reply:
Let's go with the sentence then to keep it consistent. Also because `Ilovecoding12!` will still be a strong password as per the criteria but may not exactly be strong. So, sentence it is. 

I was reading about it, I just need to click on `commit suggestion`: https://cld.wthms.co/GUI74k  and we can use the same PR to roll this out?

Reply:
Yep. :)

(Or you can just add another commit with the same text, it doesn't really matter!)

Reply:
Alright, suggestion committed.